### DISCUSSION TITLE
翻译错误

### DISCUSSION THREAD

**Discussion by nonameawa** [Light-Beacon/PCL2-NewsHomepage#195] [General]
就白罗斯大使馆2018年3月17日发文，“目前，对中国人而言，我们是来自于‘白俄罗斯’的人。‘白俄罗斯’这个词中的‘白’就是白色的白；‘俄’是个形容词，‘俄罗斯’是其含义之一；而‘罗斯’是古代巴尔特人、芬兰乌戈人和东斯拉夫人的土地上的国名……联合国正式文件和我国宪法载明，我国正式名称为‘Republic of Belarus’或‘Belarus’。从语言和语义的角度来讲，要把我国名称翻成中文的话，那就应该用‘白罗斯’这个词，即把现在的名称去掉‘俄’字。这样就不会把我国和俄罗斯联邦混为一谈。 白俄罗斯驻华大使馆谨请各位懂中文的朋友，自今日起将使用我国‘白罗斯’正确的名称。让我们说话写字正确无误！”
人民政协网亦在2018年3月21日发文《白俄罗斯为何要求改名为“白罗斯”》，详细阐述其改名背景。
目前，其大使馆已改称白罗斯，但主要政府机关还未更改，希望作为参考重置翻译。

**Comment by zkitefly** [comment]
主页有 “白俄罗斯” 的称呼吗？

**Comment by Light-Beacon** [comment]
其实我还真考虑过这个事，不过现在更通行的翻译（包括官媒啥的）是“白俄罗斯”我就用的这个

**Comment by iPodtouch6557** [comment]
……其实白俄人自己都放弃了，不知道现在政府机构怎么说

**Comment by iPodtouch6557** [comment]
……其实白俄人自己都放弃了，不知道现在政府机构怎么说
